samples/csharp/MicrosoftEduGraphSamples/Workflows/AssignmentWorkflow.cs

```diff
@@ -123,5 +123,53 @@ public async Task AssignmentsEvolvableEnumsAsync()
                 Console.WriteLine($"AssignmentsEvolvableEnumsAsync: {ex.ToString()}");
             }
         }
-    }
+
+        /// <summary>
+        /// Workflow to create Assignment resources in resource folder
```

Comment:
Workflow to create Assignment resources under the assignment resource folder

Reply:
updated to "Workflow to create Assignment resources under the assignment resource folder"